src/components/styled.js

```diff
@@ -0,0 +1,3 @@
+import styled from 'styled-components'
+
+export default styled
```

Comment:
We should also export all the helpers that come from styled-components like `injectGlobal`, `keyframes`, `css`, etc. (https://www.styled-components.com/docs/api#helpers)